source/layouts/nights.erb

```diff
@@ -7,7 +7,7 @@
     <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
     <title>LRUG Nights : <%= page_title || "solvin' crimes, drinkin' beers" %></title>
     <link href="/rss/nights/" rel="alternate" title="RSS" type="application/rss+xml" />
-    <%= stylesheet_link_tag "normalize", "nights" %>
+    <%= stylesheet_link_tag "nights" %>
```

Comment:
I moved the CSS over to SCSS and used an `@import` to include the `normalize` file. It reduces the request count by 1 :)